yb_migrate/cmd/generateReport.go

```diff
@@ -818,21 +807,10 @@ func init() {
 
 	registerCommonExportFlags(generateReportCmd)
 
-	generateReportCmd.PersistentFlags().StringVar(&outputFormat, "output-format", "html",
+	generateReportCmd.PersistentFlags().StringVar(&outputFormat, "output-format", "txt",
 		"allowed report formats: html | txt | json | xml")
 }
 
-func checkGenerateReportDirs() {
-	tempDir := exportDir + "/temp"
-	reportDir := exportDir + "/reports"
-	if startClean {
```

Comment:
@amit-yb  don't we have --start-clean flag with generateReport command now?

Reply:
We don't have the `--start-clean` flag for the genrateReport, anymore. `generateReport` overwrites the `reports/report.XXX` file every time it runs.
User can use `--start-clean` flag for the `export schema` to fetch latest schema from the DB.